istio-operator/istio-operator.spec

```diff
@@ -109,12 +109,19 @@ popd
 install -d $RPM_BUILD_ROOT/manifests
 cp -ra OPERATOR/src/github.com/maistra/istio-operator/manifests-maistra/* $RPM_BUILD_ROOT/manifests
 
+# install the templates
+mkdir -p $RPM_BUILD_ROOT%{_datadir}/istio-operator/default-templates
```

Comment:
```suggestion
TEMPLATES_DIR=$RPM_BUILD_ROOT%{_datadir}/istio-operator/default-templates
mkdir -p ${TEMPLATES_DIR}
if [[ %{community_build}  == "true" ]]; then
        cp OPERATOR/src/github.com/maistra/istio-operator/deploy/smcp-templates/maistra ${TEMPLATES_DIR}/default
else
        cp OPERATOR/src/github.com/maistra/istio-operator/deploy/smcp-templates/servicemesh ${TEMPLATES_DIR}/default
fi
cp OPERATOR/src/github.com/maistra/istio-operator/deploy/smcp-templates/base ${TEMPLATES_DIR}/default
```

Reply:
The generation of templates is in build_docker.sh which isn't executed here. This copies the relevant section over to the spec.

Reply:
Also, git_commit above should be updated to:

d6365d967cf2bc9ddd293a90bb8b3d64e34a0a0b

Reply:
this is the first time I see this feature used. wow

Reply:
@brian-avery I'm not sure I understand your changes.. the if-clause I get, but the cp afterwards overwrites it again doesn't it?